tests/api/runtime_handlers/test_kubejob.py

```diff
@@ -578,7 +578,7 @@ async def test_state_thresholds_defaults(self, db: Session, client: TestClient):
         pending_scheduled_pod.status.conditions = [
             k8s_client.V1PodCondition(type="PodScheduled", status="True")
         ]
-        pending_scheduled_pod.status.start_time = datetime.datetime.utcnow() - timedelta(
+        pending_scheduled_pod.status.start_time = datetime.datetime.now() - timedelta(
```

Comment:
why changing?

Reply:
had some issues with utcnow. changed to `datetime.now(timezone.utc)` fixed it.